packages/trader/src/Stores/Modules/Contract/contract-replay-store.js

```diff
@@ -100,8 +101,11 @@ export default class ContractReplayStore extends BaseStore {
         if (!this.switch_account_listener) return;
 
         if ('error' in response) {
+            const { code, message } = response.error;
             this.has_error = true;
             this.is_chart_loading = false;
+            this.error_message = localize(message);
```

Comment:
No need to use `localize` here, Since api will return the localized message itself.

Reply:
@reza-fs thank you for noticing this. I just fixed it